### ISSUE TITLE
If someone holds an item using an anti drop-implant, has their arm cut off and someone else picks it up it STAYS anti drop.

### ISSUE THREAD

**Issue by GuyonBroadway** [HippieStationArchive/HippieStation13-1#2920]
the situation by which this arose was i was the RD a while back, i decked myself out with augments and had an autocloner pod. Get killed by a horror form ling and got cloned. Came back, lopped off arm that my dead body was holding the gun in and found I could no longer drop it and lacked the implant so i couldn't make it "droppable".

Based on what I saw of the code it would remove the no-drop flag from cursed items.

**Comment by ArcLumin** [comment]
Yes, the way anti-drop works, it will remove curse from items, or perma no drop them. I've used it a number of times to make bombs that couldn't be put down

Setting a hardsuit to nodrop is really good for tators

**Comment by ArcLumin** [comment]
you can use this to make no drop clothing before you go sling and have completely normal clothes.

**Comment by blobbernaut** [comment]
Don't fix, I once carried around a wizard's ei nath fist after cutting it off, 7/10 was pretty good meme
